src/AvatarGroup/stories/index.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import { default as AvatarGroup, AvatarGroupProps } from '../index';
+import Avatar from '../../Avatar';
+
+export default {
+  title: 'AvatarGroup',
+  component: AvatarGroup,
+  argTypes: {
+    size: {
+      control: {
+        type: 'radio',
+        options: [64, 'large', 'default', 'small']
+      },
+      defaultValue: 64
+    }
+  }
```

Comment:
Use the new argTypes structure I gave you. Did you make sure there are no new warnings?